nodestream_akamai/siteshield.yaml

```diff
@@ -1,27 +1,26 @@
-- implementation: nodestream_akamai.siteshield:AkamaiSiteShieldExtractor
+- implementation: nodestream_akamai.siteshield:AkamaiSiteshieldExtractor
   arguments:
     base_url: !config 'base_url'
     client_token: !config 'client_token'
     client_secret: !config 'client_secret'
     access_token: !config 'access_token'
     account_key: !config 'account_key'
-  annotations:
-  - live
-  - test
 - implementation: nodestream.interpreting:Interpreter
   arguments:
     interpretations:
     - type: source_node
-      node_type: AkamaiSiteshieldConfig
+      node_type: AkamaiSiteshieldMap
       normalization:
         do_remove_trailing_dots: true
       key:
-        id: !jmespath 'id'
+        rule_name: !jmespath 'ruleName'
     - type: properties
       properties:
+        id: !jmespath 'id'
```

Comment:
Why do we want to derive the key from `rule_name` as opposed to `id`. Intuitively, seems like `id` is a better choice?

Reply:
So this was a weird one. When the key of the node was id, the relationships didnt work between Siteshield maps and properties. But, when I changed it to the rule_name, which is also unique so could work as a key, the relationships were fine. I don't know why, it doesn't make sense to me, but I suspect the issue was the Siteshield nodes were created by the relationship with the property _first_, and thus had their key as the rule_name. Then when the Siteshield pipeline ran it created new nodes with the id as they key, and I ended up with duplicate nodes. Does that track?